src/Microsoft.DotNet.Darc/Darc/Options/VirtualMonoRepo/BackflowCommandLineOptions.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using CommandLine;
+using Microsoft.DotNet.Darc.Operations;
+using Microsoft.DotNet.Darc.Operations.VirtualMonoRepo;
+
+namespace Microsoft.DotNet.Darc.Options.VirtualMonoRepo;
+
+[Verb("backflow", HelpText = "Flows code changes from the VMR back to target repositories.")]
+internal class BackflowCommandLineOptions : VmrCommandLineOptions, IBaseVmrCommandLineOptions
+{
+    [Option("additional-remotes", Required = false, HelpText =
+        "List of additional remote URIs to add to mappings in the format [mapping name]:[remote URI]. " +
+        "Example: installer:https://github.com/myfork/installer,sdk:/local/path/to/sdk")]
+    public IEnumerable<string> AdditionalRemotes { get; set; }
+
+    [Value(0, Required = true, HelpText = "Repositories to backflow in the form of NAME:PATH with mapping name and local path to the target repository. " +
+        "Path can be ommitted when ")]
```

Comment:
Think the explanation is not complete here. Also, is this `Value` attribute meant for trailing arguments, with no option?

Reply:
Yes, it's meant to be used like this: `darc vmr backflow [OPTIONS] repo1:path1 repo2:path2 ...`

```suggestion
        "Path can be ommitted when --repository-dirs is supplied.")]
```